Dockerfile

```
@@ -1,11 +1,3 @@
-
-WORKDIR /go/src
-COPY . .
-
-RUN CGO_ENABLED=0 make build/linux && chmod -R +x bin
-
-# Now copy it into our base image.
-FROM gcr.io/distroless/static-debian11
-COPY --from=build /go/src/bin/linux_amd64/ts-olly /
-CMD ["/ts-olly"]
\ No newline at end of file
+FROM gcr.io/distroless/static-debian12
```

Hadolint (2.14.0)

[warning] Always tag the version of an image explicitly

(DL3006)